AyakorK/decidim-module-extra_user_fields
# Add admin logs to features

## Description

This PR is made to add logs when an admin updates extra_user_fields options or export users because it logged very fuzzy infos before that.

## How to test it:

- Initialize your instance
- Log as an admin
- Go into BO
- Go to Settings
- Go to Extra-User-Fields
- Enable them and change some fields
- Save
- Go to Participants - Participants
- Export all with any format
- Go to your logs 
- Check that there are two logs specifying what you did.
